tests/hostfile

```diff
@@ -0,0 +1 @@
+GAO-WS5
```

Comment:
Let's remove this file - if you wish to provide an example, document this in the wiki. If you wish to keep an example in the git repo, localhost would be more appropriate.